docs/overview.md

```diff
@@ -0,0 +1,33 @@
+{% docs __overview__ %}
+
+## dbt-sythnea
+
+dbt-synthea is a project to demonstrate the power of dbt for building OMOP ETLs. Using [Synthea](https://synthetichealth.github.io/synthea/), it is possible to generate a synthetic patient data which can be transformed into the OMOP standard.
+
+### Project Structure
+
+dbt uses parameterized SQL files called [“models”](https://docs.getdbt.com/docs/build/models), which are run in a correct order by building a Directed Acyclic Graph (DAG).
+
+The models have been setup with three distinct stages:
+ - **Staging**: SQL queries that are 1:1 with the source Synthea and vocabulary tables and are used to adjust data types and column names as needed
+ - **Intermediate**: SQL queries used to perform the bulk of the joining and transformation needed to move from source tables into the OMOP CDM, with a focus on joins and transformations which may be reused several times in the final models; for example, source-to-standard concept mappings or modeling of different visit types.
+ - **Mart**: SQL queries that are 1:1 with the final OMOP tables.
+
+### Exploring the Project
```

Comment:
i think we could add all of the original template text here. i realized i didn't know most of the stuff in there after taking a look just now 🙃

Reply:
sure thing 💪